.github/workflows/pre_release.yml

```diff
@@ -0,0 +1,49 @@
+name: pre-release
+
+on:
+  create
+
+jobs:
+
+  # Make a PR to master from a new branch with changes, and delete the created one
+  update-versions:
+
+    # Only run when a release-v* branch is created, and not by drashbot
+    if: contains(github.ref, 'release-v') && !contains(github.event.sender.login, 'drashbot')
+
+    strategy:
+      matrix:
+        deno: ["1.5.1"]
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout master
+        uses: actions/checkout@v2
+        with:
+          ref: master
+
+      - name: Delete Remote Release Branch
+        run: git push --delete origin ${{ github.ref }}
+
+      - name: Install Deno
+        uses: denolib/setup-deno@v2
+        with:
+          deno-version: ${{ matrix.deno }}
+
+      - name: Bump Versions
+        run: deno run -A ./console/bumper_ci_service.ts --version=${{ github.ref }} # version passed so we know what the new release version will be
+
+      - name: Send Event to Website to Update Versions
+        run: |
+          curl -XPOST -u "${{ secrets.CI_USER_NAME }}:${{ secrets.CI_USER_PAT }}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/drashland/website/dispatches --data '{"event_type": "update_docs", "client_payload": { "module_to_update": "rhum", "version": "${{ github.ref }}" }}'
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.CI_USER_PAT }}
+          commit-message: bump versions
+          title: bump versions
+          body: This was auto-generated by GitHub Actions.
+          branch: release # Can change it to ${{ github.ref }} to be the branch name, but then we need to figure out how to stop this worjflow running  when it creates another "release-vx.x.x" branch
```

Comment:
typo: worjflow

im fine with this being `release`

Reply:
https://github.com/drashland/rhum/pull/89/commits/de4d758967f5883806063b3fd2e2bf73ecdcc52d

Reply:
the comment is there more so we know it's possible to use the ref, but it does provides problems, eg documentation on it